### ISSUE TITLE
#EXT-X-CUE-IN and #EXT-X-CUE-OUT are not dumped

### ISSUE THREAD

**Issue by aidanbarrett** [globocom/m3u8#138]
Example source m3u8:
```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:2
#EXT-X-PLAYLIST-TYPE:EVENT
#EXT-X-MEDIA-SEQUENCE:0
...
#EXT-X-CUE-OUT:30.000
#EXTINF:1.998,
1554363130757.ts
...
#EXTINF:2.001,
1554363142762.ts
#EXT-X-CUE-IN
#EXTINF:1.999,
1554363144762.ts
...
```

When it is parsed and dumped to a file the `#EXT-X-CUE-OUT` markers are replaced with `#EXT-X-CUE-OUT-CONT` with no duration and the `#EXT-X-CUE-IN` markers do not appear.

```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:2
#EXT-X-PLAYLIST-TYPE:EVENT
#EXT-X-MEDIA-SEQUENCE:0
...
#EXT-X-CUE-OUT-CONT
#EXTINF:1.998,
1554363130757.ts
...
#EXTINF:2.001,
1554363142762.ts
#EXTINF:1.999,
1554363144762.ts
...
```

**Comment by mauricioabreu** [comment]
Thank you for opening this issue. I will investigate what is happening, and provide a fix if necessary.

**Comment by mauricioabreu** [comment]
I found the related bug. Fixing it is not so easy so it will take some time to fix.
I hope to deliver the fix in the next version (`0.3.9`)

**Comment by aidanbarrett** [comment]
Awesome! Thanks for that
